src/v2/api/controllers/GroupController.ts

```diff
@@ -172,7 +173,12 @@ export class GroupController {
   @ApiBadRequestResponse({
     description: `\n
     InvalidGroupIdException: 
```

Comment:
Change the name of exeption 
![image](https://github.com/fictadvisor/fictadvisor-api/assets/116638814/26e5608f-12df-45a5-8cca-4208612012d8)